### PR TITLE
[CPU] added cpu specific properties to supported_properties report

### PR DESCRIPTION
**Pull request by antonvor**

### CODE REVIEW

**Comment by antonvor** [comment]
@dmitry-gorokhov @ilya-lavrenov take a look please

File: src/plugins/intel_cpu/src/plugin.cpp [hunk 1]
```diff
@@ -637,6 +640,11 @@ Parameter Engine::GetMetric(const std::string& name, const std::map<std::string,
     } else if (name == ov::caching_properties) {
         std::vector<ov::PropertyName> cachingProperties = { ov::device::full_name };
         return decltype(ov::caching_properties)::value_type(cachingProperties);
+    } else if (name == ov::intel_cpu::denormals_optimization) {
+        return decltype(ov::intel_cpu::denormals_optimization)::value_type(engConfig.denormalsOptMode == Config::DenormalsOptMode::DO_On);
+        return engConfig.denormalsOptMode;
+    } else if (name == ov::intel_cpu::sparse_weights_decompression_rate) {
+        return decltype(ov::intel_cpu::sparse_weights_decompression_rate)::value_type(engConfig.fcSparseWeiDecompressionRate);
```
Comment (ilya-lavrenov) [inline src/plugins/intel_cpu/src/plugin.cpp:647]: do we have tests for it?

Comment (dmitry-gorokhov) [reply src/plugins/intel_cpu/src/plugin.cpp:647]: @antonvor Take this PR as reference for tests: https://github.com/openvinotoolkit/openvino/pull/16106. The only difference the test shouldn't be shared, but CPU specific.

File: src/plugins/intel_cpu/src/plugin.cpp [hunk 2]
```diff
@@ -637,6 +640,11 @@ Parameter Engine::GetMetric(const std::string& name, const std::map<std::string,
     } else if (name == ov::caching_properties) {
         std::vector<ov::PropertyName> cachingProperties = { ov::device::full_name };
         return decltype(ov::caching_properties)::value_type(cachingProperties);
+    } else if (name == ov::intel_cpu::denormals_optimization) {
```
Comment (EgorDuplensky) [inline src/plugins/intel_cpu/src/plugin.cpp:643]: Should we also return these properties from exec_network (CompiledModel)?

**Comment by EgorDuplensky** [comment]
Proceeded in scope of #16720

**Pull request closed without merging**